src/main/java/apoc/export/cypher/MultiStatementCypherSubGraphExporter.java

```diff
@@ -249,14 +251,36 @@ private void appendRelationship(PrintWriter out, Relationship rel, Reporter repo
         out.print("MATCH ");
         out.print(nodeLookup("n1", rel.getStartNode()));
         out.print(", ");
+        long props = 0;
         out.print(nodeLookup("n2", rel.getEndNode()));
-        out.print(" CREATE (n1)-[:");
```

Comment:
this could be a `typeExport.createRelationship()` up to and including: formatPropertiesForRelationship